### DISCUSSION TITLE
Servers off?

### DISCUSSION THREAD

**Discussion by metroidteam** [revoxhere/duino-coin#557] [General]
Servers off?it won't let me connect to continue mining, the servers don't work?

**Comment by revoxhere** [comment]
Should be working fine now.

**Reply by Sasank-cyber** [reply]
what about pc mining server please reply sir

**Reply by Sasank-cyber** [reply]
is pc mining stopped

**Comment by metroidteam** [comment]
Thank you, I think it is a project with a future, I discovered it
yesterday, the more I mine I will make more donations for the cause, I had
a. Unused arduino mega, I see that it is going very well congratulations
and I will continue to support
